src/device/pf_cmina.c

```diff
@@ -1274,6 +1274,35 @@ int pf_cmina_remove_all_data_files (const char * file_directory)
    return 0;
 }
 
+/**
+ * Check if a time interval has passed
+ *
+ * @param now              In:    Current time in microseconds
+ * @param previous         In:    Previous time in microseconds
+ * @param period           In:    Time interval length in 1/32 of millisecond.
+ *                                For 1 millisecond use 32
+ *                                    2 milliseconds use 64 etc
+ * @param factor           In:    Multiplication factor for the timeout.
+ *                                Use 1 if the given time interval length should
+ *                                be used, or 3 if we should return true after
+ *                                3 time interval lengths etc
+ * @return  true if the time interval has passed
+ *          false if it not has passed
+ */
+bool pf_cmina_has_timed_out (
+   uint32_t now,
```

Comment:
Maybe start_time_us and end_time_us instead of now and previous

Reply:
Done